crates/derive/src/stages/batch_queue.rs

```diff
@@ -261,6 +270,9 @@ where
         if drop {
             self.prev.flush();
```

Comment:
So do you now also flush pre-holocene?

Reply:
Good catch - no, we shouldn't be.

Reply:
does this also prevent flushing in the batchqueue itself?

Reply:
Yes it does - is that incorrect?

Reply:
It also needs to be flushed if a batch is found to be invalid (`drop`).